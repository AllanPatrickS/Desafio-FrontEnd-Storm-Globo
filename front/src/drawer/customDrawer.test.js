import { cleanup } from '@testing-library/react';
import { create } from "react-test-renderer";
import Drawer from './customDrawer';

describe('Testing drawer', () => {

  it('renders correctly', () => {
    const wrapper = create(<Drawer row={{
      _id: 'teste',
      username: 'test',
      email: 'email@teste.com',
      createdAt: '20/20/2020',
      updatedAt: '20/20/2020',
      rules: 0,
      status: true
    }} />)
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  afterEach(function () {
    cleanup();
  });

});