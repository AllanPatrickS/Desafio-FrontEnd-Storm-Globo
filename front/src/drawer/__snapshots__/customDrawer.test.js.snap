// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing drawer renders correctly 1`] = `null`;
